vagrant/provisioning/roles/arkcase-pre-upgrade/tasks/main.yml

```diff
@@ -20,6 +20,11 @@
   set_fact:
     config_backup_folder_path: "{{ root_folder }}/data/arkcase-home/.arkcase-backup-{{ ansible_date_time.iso8601 }}"
 
+- name: Delete all but last 5 backups
+  become: yes
+  become_user: arkcase
+  shell: ls -atp | grep ".arkcase-backup*"| tail -n +6 | xargs -d '\n' -r rm -rf
```

Comment:
@dzdimov , what will happen if the tail output is empty, like when there are fewer than 5 previous backups?

Reply:
Also, the following command (which creates a new backup of the folder) specifies the starting folder "{{ root_folder }}/data/arkcase-home/", I think for consistency we should have the same starting folder in this command.

Reply:
No issue when there are fewer than 5 previous backups. We tested it.
We fixed the issue with starting folder. Thanks for that one.